src/fat.rs

```diff
@@ -232,6 +232,17 @@ fn name_to_str(input: &str, output: &mut [u8]) {
             break;
         }
     }
+
+    // Fixup for Director or file name, when the length of its name and extend name equals 11. There maybe no
```

Comment:
nit: typo here - I think you mean directory. Could you also wrap you comment to 80 characters